pkg/awsutils/vpc_ip_resource_limit.go

```diff
@@ -144,7 +144,7 @@ var InstanceNetworkingLimits = map[string]InstanceTypeLimits{
 	"i3en.large":    {ENILimit: 3, IPv4Limit: 10},
 	"i3en.metal":    {ENILimit: 15, IPv4Limit: 50},
 	"i3en.xlarge":   {ENILimit: 4, IPv4Limit: 15},
-	"inf1.24xlarge": {ENILimit: 15, IPv4Limit: 30},
+	"inf1.24xlarge": {ENILimit: 11, IPv4Limit: 30},
```

Comment:
Curious why the limits changed for this instance type?

Reply:
Looks like https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html needs to be updated. This is automatically generated from aws cli describe instances - https://github.com/aws/amazon-vpc-cni-k8s/blob/615a2a6b6690abc02e043f13ebd16bb0f979be56/scripts/gen_vpc_ip_limits.go#L68

Reply:
Got it, thanks!